pkgs/development/python-modules/shlib/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, braceexpand
+, inform
+}:
+
+buildPythonPackage rec {
+  pname = "shlib";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "KenKundert";
+    repo = "shlib";
+    rev = "v${version}";
+    sha256 = "sha256-2fwRxa64QXKJuhYwt9Z4BxhTeq1iwbd/IznfxPUjeSM=";
+  };
+
+  pythonImportsCheck = [ "shlib" ];
+  propagatedBuildInputs = [ braceexpand inform ];
+
+  meta = with lib; {
+    description = "shell library";
+    homepage = "https://nurdletech.com/linux-utilities/shlib";
```

The URL give me a 404.

```suggestion
    homepage = "https://github.com/KenKundert/shlib";
    changelog = "https://github.com/KenKundert/shlib/releases/tag/v${version}";
```